content/doc/book/using/best-practices.adoc

```diff
@@ -0,0 +1,177 @@
+---
+layout: section
+title: Best Practices
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+Continuous Integration (CI) with automated test execution and trends has changed the way companies look at build management, release management, deployment automation, and test orchestration.
+This section describes best practices to provide executives, business managers, software developers, and architects with a better sense of what Jenkins can contribute throughout the project lifecycle.
+
+== Automate job definition
+
+Jenkins can automatically create, update, and delete jobs based on the repositories it detects in your software configuration management system.
+Structure your job definitions to get the most benefit from automatic job management by Jenkins.
+
+There are multiple alternatives for automatic job management, including:
+
+* <<Use organization folders>> - create, update, and delete multibranch Pipeline folders and Pipeline jobs automatically (preferred)
```

Comment:
The <<Use organization folders>> markup is required as a link to the later heading in the document.  I liked it because I thought it was more action oriented.  I think that a best practices page is the right place to be more explicit on actions that the reader should take.